pkg/scheduler/apis/config/types.go

```diff
@@ -131,3 +143,72 @@ type KubeSchedulerLeaderElectionConfiguration struct {
 	// LockObjectName defines the lock object name
 	LockObjectName string
 }
+
+// Plugins include enabled plugins and disabled plugins, when enabled, the list of plugins for
+// a particular extension point are the only ones enabled. If an extension point is
+// omitted from the config, then the default set of plugins is used for that extension point.
+type Plugins struct {
+	// EnabledPlugins specifies plugins that should be enabled in addition to default plugins.
```

Comment:
either way can work, but is it more natural to slice by enabled/disabled as the first level, or by the extension point?

```yaml
enabledPlugins:
  queueSort:
  - foo
  - bar
  preFilter:
  - foo
  - bar
disabledPlugins:
  queueSort:
  - baz  
  preFilter:
  - baz  
```

vs 

```yaml
queueSort:
  enabled:
  - foo
  - bar
  disabled:
  - baz
preFilter:
  enabled:
  - foo
  - bar
  disabled:
  - baz
```

when trying to answer the question "what plugins will run for a given extension point?" the latter seems easier to reason about to me.

Reply:
I agree that the latter approach is easier to read.